resources/views/components/layout.blade.php

```diff
@@ -16,7 +16,11 @@
 </head>
 
 <body class="h-full">
+    <button id="backToTopBtn" class="hidden fixed top-20 left-1/2 z-10" style="transform: translateX(-50%, -50%);">
```

Comment:
Pour rester coherent avec les classes, j'utiliserai kebab-case pour les id aussi back-to-top-btn